plugins/parsers/grok/parser.go

```diff
@@ -57,7 +58,7 @@ var (
 	//     %{IPORHOST:clientip:tag}
 	//     %{HTTPDATE:ts1:ts-http}
 	//     %{HTTPDATE:ts2:ts-"02 Jan 06 15:04"}
-	modifierRe = regexp.MustCompile(`%{\w+:(\w+):(ts-".+"|t?s?-?\w+)}`)
+	modifierRe = regexp.MustCompile(`%{\w+:(\w+|):(ts-".+"|t?s?-?\w+)}`)
```

Comment:
Let's just require a name that isn't used, similar to how timestamp works.  This will make the code even simpler and we won't need to explain the new form.